pod/video/views.py

```diff
@@ -47,6 +47,12 @@
 
 TODAY = date.today()
 VIDEOS = Video.objects.filter(encoding_in_progress=False, is_draft=False)
+try:
+    for x in VIDEOS:
+        if not x.encoded:
+            VIDEOS = VIDEOS.exclude(id=x.id)
```

Comment:
pourquoi ne pas utiliser exclude(pk__in=[vid.id if vid.encoded for vid in VIDEOS]) ou un truc comme ca ? et idem, merci de remplacer x par vid. ok pour le try catch, verifions si les tests passent :)

Reply:
Fait